get_github_repos.py

```diff
@@ -6,14 +6,14 @@
 
 TOKEN = os.environ.get("GITHUB_TOKEN")
 
-exclude = {line.split(" ", 1)[0] for line in Path("exlude_repos.txt").read_text().splitlines()}
+exclude = {line.split(" ", 1)[0] for line in Path("exclude_repos.txt").read_text().splitlines()}
 
 
 def get_repos():
     from github import Github
 
     github = Github(TOKEN)
-    search = github.search_code("mkdocstrings+filename:mkdocs.yml")
+    search = github.search_code("mkdocstrings filename:mkdocs.yml")
```

Comment:
Ah, I was wondering why it didn't work anymore. After I created the mkdocstrings org on GitHub, the script was returning the org only, nothing else.